lab2.js

```diff
@@ -43,26 +43,55 @@ function assert(expression, failureMessage) {
     third hour      |    3/hour        |        3         |
     fourth hour     |    4/hour        |        4         |
 
- TODO: First, make a constructor function, called Blob, that makes blobs.
+ TODO: First, make a constructor function, called Blob, that makes blobs.*/
+ function Blob (personsConsumed, rateOfConsumption) {
+  this.personsConsumed = personsConsumed;
+  this.rateOfConsumption = rateOfConsumption;
+  this.totalTime = 0;
+ }
 
- TODO: Next, create an instance of Blob named blob.
+ //TODO: Next, create an instance of Blob named blob.//
+ var blob = new Blob();
 
- TODO: Then, use a loop to calculate how long it took the blob to finish
- with Dowington. */
+ //TODO: Then, use a loop to calculate how long it took the blob to finish
+ //with Dowington.
+ var dowington = 1000;
+
+ function rateOfBlob(hours) {
+  var peopleEaten = hours + 1;
+  return peopleEaten;
+ }
+
+ var blob = new Blob(0,1);
+
+ for (var rateOfConsumption = 1; rateOfConsumption <= 1001; rateOfConsumption++) {
+  this.totalTime += (60 / rateOfConsumption);
+ }
+
+ blob.conversion = function() {
+  var result = (this.totalTime / 60);
+  var hours = Math.floor(result);
+  return (hours);
+ };
 
 
 var hoursSpentInDowington; // TODO: assign me the value of the
                            // above calculation (how long it took
                            // the blob to eat Dowington)
+blob.conversion();
```

Comment:
missing the piece about assigning the value of the above calculation to the var hoursSpentInDowington. Your asserts pass, but they pass because you're comparing undefined to undefined.